### ISSUE TITLE
can your share your detailed environment

### ISSUE THREAD

**Issue by WangXiequn** [Chaos96/fourierft#7]
can you post your conda environment of thi sproject?

**Comment by WangXiequn** [comment]
by the way, it seems there are two argparser parsing different args which casues issue Parsing args
usage: SFT.py [-h] [--model_name_or_path MODEL_NAME_OR_PATH] [--dataset DATASET] [--bs BS] [--num_epochs NUM_EPOCHS] [--n_frequency N_FREQUENCY] [--lr LR] [--max_length MAX_LENGTH]
              [--weight_decay WEIGHT_DECAY] [--warm_step WARM_STEP] [--train_ratio TRAIN_RATIO] [--scale SCALE] [--width WIDTH] [--fc FC] [--share_entry] [--set_bias] [--seed SEED]
              [--entry_seed ENTRY_SEED]
SFT.py: error: unrecognized arguments: --model_tag llama-2-7b how to solve it?

**Comment by zqgao22** [comment]
> by the way, it seems there are two argparser parsing different args which casues issue Parsing args `usage: SFT.py [-h] [--model_name_or_path MODEL_NAME_OR_PATH] [--dataset DATASET] [--bs BS] [--num_epochs NUM_EPOCHS] [--n_frequency N_FREQUENCY] [--lr LR] [--max_length MAX_LENGTH] [--weight_decay WEIGHT_DECAY] [--warm_step WARM_STEP] [--train_ratio TRAIN_RATIO] [--scale SCALE] [--width WIDTH] [--fc FC] [--share_entry] [--set_bias] [--seed SEED] [--entry_seed ENTRY_SEED]` SFT.py: error: unrecognized arguments: --model_tag llama-2-7b how to solve it?

Hi, there were some argument errors that caused bugs, which may not be related to the environment. You can now try running SFT for LLaMA again.

Best.
